parser/parser.go

```diff
@@ -3166,6 +3179,168 @@ func (p *parser) parseDecl(sync map[token.Token]bool) ast.Decl {
 // ----------------------------------------------------------------------------
 // Source files
 
+func (p *parser) entryPoint(pkgname string) string {
```

Comment:
don't need this. can always return "main" (We can let `cl` to change mock function name to be what we want).

Reply:
remove, always set "main"